includes/admin/class-ur-admin-user-list-manager.php

```diff
@@ -147,6 +148,42 @@ public function pending_users_notices() {
 		}
 	}
 
+	/**
+	 * Display a notice to admin notifying the new registered users.
+	 */
+	public function new_users_notices() {
+
+		$screen    = get_current_screen();
+		$read_time = get_option( 'user_registration_users_listing_viewed' );
+		$now       = date( 'Y-m-d h:i:s' );
+
+		if ( ! $read_time ) {
+			update_option( 'user_registration_users_listing_viewed', $now );
+			$read_time = $now;
+		}
+
+		if ( 'users' === $screen->id ) {
+			update_option( 'user_registration_users_listing_viewed', $now );
+		}
+
+		$user_args  = array(
+			'meta_key'    => 'ur_form_id',
+			'count_total' => true,
+			'date_query'  => array(
+				array(
+					'after'     => $read_time,
+					'inclusive' => false,
+				),
+			),
+		);
+		$user_query = new WP_User_Query( $user_args );
+		$user_count = $user_query->get_total();
+
+		if ( 0 < $user_count && 'users' === $screen->id ) {
+			echo '<div id="new-user-live-notice" class="notice notice-success is-dismissible"><p><strong>' . __( 'User Registration:', 'user-registration' ) . '</strong> ' . $user_count . ( ( $user_count === 1 ) ? __( ' new User', 'user-registration' ) : __( ' new Users', 'user-registration' ) ) . __( ' registered.', 'user-registration' ) . '</p></div>';
```

Comment:
Over here for singular and plural sentence utilize `_n`, my friend. This is not the WordPress way overall.

Reply:
Thanks for the Suggestion. I will do so. 😃